am-lib/src/test/java/uk/gov/hmcts/reform/amlib/AddServiceDefaultRolesTest.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.amlib;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+class AddServiceDefaultRolesTest {
+    private final DefaultRoleSetupImportService defaultRoleSetupImportService = new DefaultRoleSetupImportService(
```

Comment:
I like short names that are unique within the context. I think simple name as `service` would do well here.